### ISSUE TITLE
Not working on bamboo doors.

### ISSUE THREAD

**Issue by Wolfie713** [Brianetta/StickyLocks#19]
Haven't tested it on Cherry Blossom doors, but those might be affected as well.  Using MC 1.20.4.

**Comment by Brianetta** [comment]
This is usually just a matter of adjusting the configuration. I will update StickyLocks' default values when I get a moment this week.

In the short term you can add missing values to your `config.yml` from [Spigot's Materials page](https://hub.spigotmc.org/javadocs/spigot/org/bukkit/Material.html).

You'll be looking at all the wood types for things like buttons, fence gates, trap doors, pressure plates and so on. When there's a new wood type, what I do is search for one of the old wood types (like `BIRCH_`) and copy the things I find with the new types (`BAMBOO_` and `CHERRY_` and `MANGROVE_` in this case).

If you have customised your `config.yml` you will still have to edit it yourself once I've updated the plugin. If you haven't, you can delete `config.yml` and let the new version copy its default one out for you.

Looks like there are a bunch of things that have been added in recent years, like cauldron variants and new redstone thingies. I'll take a look at all of this.

**Comment by Wolfie713** [comment]
Looking at the list, would SL work on chest boats?

**Comment by Brianetta** [comment]
No, it won't. SL only works on placed blocks, not entities. We can't lock hopper/furnace/chest minecarts or donkeys/mules/llamas either.

**Comment by Wolfie713** [comment]
I had a feeling that it not being a static item would make a difference.  Drat. lol
